### ISSUE TITLE
Docs contain broken links

### ISSUE THREAD

**Issue by digitalcraftsman** [rdwatters/hugo-docs-concept#21]
I used a [broken-link checker](http://www.brokenlinkcheck.com/) to find such and it turns out there are quite a few links that cause an 404 error.

This might happen because 

- external resources moved/changed/got deleted
   - e.g. from the showcase or articles from the press section
- the edit-this-page button refers to pages that currently don't exist in the spf13/hugo repo
- some tags are linked at the top right on a page that don't exist
   - e.g. the "read more" tag on [this page](https://hugodocs.info/content-management/summaries/)

Here's a list of broken links that cause an 404 error. It's stupid that's not possible to link certain lines in a Markdown file in source code files. Hence I've to guide you differently:

- Link "called directly in the Hugo docs" on [https://hugodocs.info/templates/files/#readfile-example-add-a-project-file-to-content] (at the bottom)

**Comment by rdwatters** [comment]
Ah yes, that link *should* be broken, actually, although I needed to update it.

My assumption is that the site will live in the spf13 official repo, so when I was creating and editing content, I linked to where the source files *will* be in the future. So the example of "https://github.com/spf13/hugo/blob/master/docs/content/templates/files.md" doesn't exist, but hopefully it will in the future:) 

That said, I'm sure there are a ton others, so please feel free to list and send along. Thanks!

**Comment by digitalcraftsman** [comment]
> So the example of "https://github.com/spf13/hugo/blob/master/docs/content/templates/files.md" doesn't exist, but hopefully it will in the future:)

Thanks for the confirmation. I could also have been an artifact left over from the reorganization or the content.

**Comment by rdwatters** [comment]
@digitalcraftsman I ran the same link checker. Originally had a few hundred broken links, but now it's down to roughly 35 broken links, all of which are not actually broken but are just links to different GH repos. 

I also went through the very boring process of checking all the articles in "Press and Articles" and removed all the dead sites or links I could find. Still need to switch this over to a data file eventually per #19 Thanks again!
